dev/airflow-pr

```diff
@@ -590,15 +642,20 @@ def main(pr_num, local=False):
     while raw_input("\n%s (y/n): " % pick_prompt).lower() == "y":
         merged_refs = merged_refs + [cherry_pick(pr_num, merge_hash, latest_branch)]
 
-    continue_maybe("Would you like to update an associated JIRA?")
+    continue_maybe("Would you like to update associated JIRA issues?")
     jira_comment = "Issue resolved by pull request %s\n[%s/%s]" % (pr_num, GITHUB_BASE, pr_num)
+
     jira_ids = re.findall("AIRFLOW-[0-9]{1,6}", title)
-    if not jira_ids:
-        resolve_jira_issue(
```

Comment:
It looks like this was dropped. Was this intentional? Now, if no JIRA is found, it just skips over asking.